config/templates/cloud-config-worker

```diff
@@ -466,8 +465,6 @@ write_files:
            done;
            echo done.'
 
-      sudo rkt rm --uuid-file=/var/run/coreos/decrypt-tls-assets.uuid
```

Comment:
@redbaron Sorry missed the context on my side this time. I'd like to leave `rkt rm` as many as possible for system pods like decrypt-tls-assets to provide us chances to immediately notice issues like https://github.com/coreos/kube-aws/pull/199#issuecomment-271000614. Getting rid of `rkt rm`s does defer but doesn't fix an issue like that.

I'm ok with `rkt gc` doing their jobs not on system pods(created via kube-aws) but rather user pods.